### PR TITLE
[Independent-Publisher] certify-connector

### PR DESCRIPTION
**Pull request by troystaylor**
---
When submitting a connector, please check the following conditions for your PR.

- [X] I attest that the connector works and I verfied by deploying and testing all the operations.
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`.

### CODE REVIEW

File: independent-publisher-connectors/HubSpot CMS/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,569 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "HubSpot CMS",
+    "description": "HubSpot CMS community connector.",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/HubSpot CMS/apiDefinition.swagger.json:5]: @troystaylor Can you please provide valid description as it should describe concisely the main purpose and value offered by your connector.

Documentation - [Description of connector](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#description)

File: independent-publisher-connectors/HubSpot CMS/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,569 @@
...
+              "properties": {
+                "campaign": {
+                  "title": "Campaign",
+                  "type": "string",
+                  "description": "campaign"
+                },
+                "campaign_name": {
+                  "title": "Campaign name",
+                  "type": "string",
+                  "description": "campaign_name"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/HubSpot CMS/apiDefinition.swagger.json:103]: @troystaylor Can you please provide valid description valid with out grammatical values.

Documentation - [Description of properties](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)

Applies to all the similar values for the properties in swagger.

File: independent-publisher-connectors/HubSpot CMS/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+          "displayName": "Use HubSpot OAuth 2.0",
+          "description": "Log in using HubSpot app client ID and secret."
+        },
+        "properties": {
+          "connectionParameters": {
+            "token": {
+              "type": "oauthSetting",
+              "oAuthSettings": {
+                "identityProvider": "oauth2",
+                "clientId": "7c3afd4c-a688-4f38-a0d9-6e4a1a1b60ed",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/HubSpot CMS/apiProperties.json:20]: @troystaylor Please update the client id with dummy value, like below:

`"clientId": "[[DUMMY]]"`

File: independent-publisher-connectors/HubSpot CMS/settings.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (sriyen-msft) [inline independent-publisher-connectors/HubSpot CMS/settings.json:1]: @troystaylor Please delete `icon.png` and `settings.json` files as these are not required for opensourcing your connector.

File: independent-publisher-connectors/HubSpot CRM/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,6272 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Default title",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/HubSpot CRM/apiDefinition.swagger.json:4]: @troystaylor Can you please provide valid title for connector.

Documentation - [Title](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#title)

File: independent-publisher-connectors/HubSpot CRM/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,6272 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Default title",
+    "description": "HubSpot\u2019s CRM is a system for managing customer relationships and storing data. The CRM custom connector provides access to CRM objects or groups of objects, which are represented as a map of property names to values.",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/HubSpot CRM/apiDefinition.swagger.json:5]: @troystaylor Please remove all the unicodes from connector artifacts as unicodes may not be supported in all the Power Automate pages

File: independent-publisher-connectors/HubSpot CRM/apiDefinition.swagger.json [hunk 7]
```diff
@@ -0,0 +1,6272 @@
...
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "properties": {
+                  "type": "object",
+                  "properties": {
+                    "about_us": {
+                      "title": "About usAbout Us",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/HubSpot CRM/apiDefinition.swagger.json:90]: @troystaylor Is it typo ?

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@troystaylor The feedback I have provided for first connector can be applied to other connectors in the PR.

**Comment by mamurshe** [comment]
Please submit separate PR per connector

**Comment by sriyen-msft** [comment]
@troystaylor Kindly split the PR into multiple PRs per connector which eases to review. Thanks

**Pull request closed without merging**